DOC-2276 specify scope of externally generated JWT token (4.1) (#501)
tigergraph/server-docs: modules/user-access/pages/jwt-token.adoc

=== Non-Interactive
Users can configure JWT token non-interactively by using the `gadmin` CLI tool by running one of these commands for RSA or HMAC.

[NOTE]
====

> The JWT token generated in this section is only supported by restpp side.

Reviewer: refactor to:
The JWT token issued by a third party is limited to authentication on the RESTPP side exclusively. To enable authentication for both GSQL and RESTPP services, it is recommended to generate the token using the GSQL API service.

> The JWT token issued by a third party is limited to authenticate on the RESTPP side exclusively. To enable authentication for both GSQL and RESTPP services, it is recommended to generate the token using the GSQL API service.

Reviewer 2: ```suggestion
The JWT token issued by a third party is limited to authentication for RESTPP services only.
To enable authentication for both GSQL and RESTPP services, it is recommended to generate the token using the GSQL API service.
```
